spec/index.html

```diff
@@ -1009,7 +1105,7 @@ <h4>Representation of Literals</h4>
       <p>Some concrete syntaxes MAY support
         <dfn data-lt="simple literal" class="export">simple literals</dfn> consisting of only a
         <a>lexical form</a> without any <a>datatype IRI</a>, <a>language tag</a>, or <a>base direction</a>.
-        Simple literals are syntactic sugar for abstract syntax
+        Simple literals are syntactic sugar for <a>abstract syntax</a>
```

Comment:
```suggestion
        Except where specified otherwise by a concrete syntax
        (also known as a serialization format) specification,
        as with the syntactic sugar of `a`, `true`, and `false` in Turtle,
        simple literals are syntactic sugar for <a>abstract syntax</a>
```

Reply:
I don't know how to read this.

simple literals  - "like this" -  are strings as mentioned at 1107. Lexical forms have quoted round them.

Turtle `a` or `true` aren't a lexical forms. They are keywords.

Reply:
line 1011/1107 says its about lexical forms - i.e. quoted strings.

 `a`, `true`, and `false` are keywords of the grammar - no quotes. The original text is fine.

I woudl chnage 1009/1105 and remove the `MAY` -- "Some concrete syntaxes support ..." which is a statement of fact.